experimental/null_safety/null_unsafe_app/test/widget_test.dart

```diff
@@ -0,0 +1,30 @@
+// This is a basic Flutter widget test.
```

Comment:
I don't know that this would pass. You could either cut it entirely or copy over the one from the other app.

Reply:
Removed